### PR TITLE
CS: general fixes

### PR DESCRIPTION
**Pull request by keradus**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | ? |
| Fixed tickets | N/A |
| License | MIT |
| Doc PR | N/A |

After this PR the whole 2.3 version will be valid when running PHP CS Fixer.
It is also first step to apply PHP CS Fixer on next version.

To keep fabbot.io happy ;)

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/attributes.html.php [hunk 1]
```diff
@@ -1 +1 @@
-<?php echo $view['form']->block($form, 'widget_attributes') ?>
+<?php echo $view['form']->block($form, 'widget_attributes');
```
Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/attributes.html.php]: This one is perfectly valid and I won't change it. The same goes for all PHP templates below. I think there is a big difference between PHP classes (where we should avoid the ending `?>`) and PHP templates, where you mostly have a bunch of PHP code within HTML/text, ... In the case of PHP templates, having all PHP code properly wrapped is important. I know that this PHP template and some others in Symfony are a bit extreme and edge cases, but nonetheless, they are templates and should be treated as such.

There are many ways to make the difference, but I think the easiest one is when the file only contains one big chunk of PHP and nothing else (big chunk meaning that we are talking about a significant amount of PHP).

Thoughts?

Comment (keradus) [inline src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/attributes.html.php]: I have same dilemmas therefore I have separated template changes into second commit.

When a template have `T_INLINE_HTML` we know it is a template and don't do changes like that. But here - we don't know that.  Big PHP chunk may be in non-template file too, and how big is big enough...

You are against removing `?>` in templates, are you also against adding missing `;` or new line for control structure block?

What about leaving only first commit ?

Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/attributes.html.php]: First commit is fine by me. :+1:

Comment (keradus) [inline src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/attributes.html.php]: Done ;)

**Comment by fabpot** [comment]
Thank you @keradus.

**Pull request merged**